language/testing-infra/transactional-test-runner/src/vm_test_harness.rs

```diff
@@ -203,7 +206,11 @@ impl<'a> MoveTestAdapter<'a> for SimpleVMTestAdapter<'a> {
                     format_vm_error(&e)
                 )
             })?;
-        Ok((None, serialized_return_values))
+
+        Ok((
+            Some(format!("ChangeSet: {:?}\n", change_set)),
```

Comment:
We should really do some custom display here, feels a bit weird (and unlike the testing framework) to give just the raw debug dump

Reply:
Any suggestions for what we should dump here?